doc/datamodel/compartments.md

```diff
@@ -0,0 +1,53 @@
+Compartment report data model {#data_compartments}
+============
+
+## Prerequisites
+
+A _neuron_ is a single nerve cell simulated, and connected with thousands of
+other neurons. It is constructed from a morphological skeleton aka _morphology_.
+This skeleton has a tree-like structure originating at the soma, the cell body
+of each neuron. Each neuron has a unique _GID_ (global identifier). Every neuron
+represents an instance of a certain morphology in 3D space which is link in the
+MVD circuit description.
+
+The branches of the tree are formed by _sections_ which are subdivided into
+smaller pieces. They start at the soma and eventually branch into multiple
+sections in a tree-like structure.
+
+During the reconstruction from real neurons, each section is formed by a number
+of _segments_. Each segment has two endpoints in 3D space, as well as a radius
+for each endpoint. These segments consequently represent tubelets in 3D space.
+
+For the simulation, each section is decomposed into m _compartments_, which have
+a n : m mapping to segments. For each compartment, various observables are
+simulated and may be reported in a CompartmentReport.
+
+## Meta data
+
```

Comment:
You always start enumerations with lowercase and omit dots at the end of each line. Is that proper English punctuation? Because it looks strange to me, at least when each item is a sentence.